src/quoteoftheday/routes.py

```diff
@@ -41,6 +39,15 @@ def index():
 
```

Comment:
NIT: can we rename `show_greeting` to something like `greeting_message`. Currently, it sounded more like a boolean.

Reply:
This demo was originally a boolean. It is text now. Updated the name.